unstructured/documents/elements.py

```diff
@@ -553,4 +561,12 @@ class Footer(Text):
     "Table": Table,
     "Header": Header,
     "Footer": Footer,
+    "Caption": Text,
```

Comment:
should this not map to `FigureCaption` ?

Reply:
You're right! Updating...